app/kubernetes/views/configurations/configurations.html

```diff
@@ -5,6 +5,19 @@
 <kubernetes-view-loading view-ready="ctrl.state.viewReady"></kubernetes-view-loading>
 
 <div ng-if="ctrl.state.viewReady">
+  <information-panel title-text="Advanced deployment" ng-if="ctrl.state.isAdmin">
```

Comment:
@Xuorg should we share this template with the other views ?

Reply:
@sbusso We could have. Do you want me to make a template and use it everywhere ?